packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -15,11 +15,11 @@ function setExtraConsole {
     source (loop0)/etc/os-release
     set baseExtraConsole="console=ttyS0"
     # rpi
-    if regexp "arm-rpi" $KAIROS_FLAVOR; then
+    if regexp "arm64-rpi" $KAIROS_ARTIFACT; then
```

Comment:
I think we could drop the arm stuff in here and orin, no way its gonna be x86 no?